_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -180,6 +180,8 @@ function page_admin_scripts() {
 			wp_enqueue_script( 'jp-tracks', '//stats.wp.com/w.js', array(), gmdate( 'YW' ), true );
 		}
 
+		wp_set_script_translations( 'react-plugin', 'jetpack', JETPACK__PLUGIN_DIR . '/languages/json/' );
```

Comment:
@akirk I don't quite understand `JETPACK__PLUGIN_DIR . '/languages/json/'`. Do the translations need to live within the Jetpack plugin? Isn't the default good enough? Is this because we want to use the translations we pull manually on each release instead of the ones provided by language packs? If we do that, does this mean that folks will need to wait to get new strings until each new release?

Reply:
I might have written this code at a time where this was not an option yet. Clearly the path should be removed if we use language packs.

Reply:
Ok, thanks! Done in 56c92cba1c77f4018eb402b253c59a630fcf1c6c